### PR TITLE
Implement the Contact email address page - RIP-119 

### PR DESCRIPTION
**Pull request by autotelik**
https://eaflood.atlassian.net/browse/RIP-119

Create Form, view and locale information for the Contact Email address page

### CODE REVIEW

File: app/forms/flood_risk_engine/steps/correspondence_contact_email_form.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      class << self
+        private
+        def new(model, enrollment = nil)
+          super(model, enrollment)
+        end
+      end
+
+    end
+  end
+end
```
Comment (woodpigeon) [inline app/forms/flood_risk_engine/steps/correspondence_contact_email_form.rb]: @autotelik Had a go at the conditional validation and this works for me:
```ruby
validates :email_address,
        presence: {
          message: I18n.t("#{locale_key}.errors.email_address.blank")
        },
        email_format: {
          allow_blank: true,
          message: I18n.t("#{locale_key}.errors.email_address.format")
        }

      validates :email_address_confirmation,
        presence: {
          message: I18n.t("#{locale_key}.errors.email_address_confirmation.blank"),
          unless: ->(form) { form.errors.any? }
        }

      validates :email_address,
        confirmation: {
          message: I18n.t("#{locale_key}.errors.email_address_confirmation.format"),
          unless: ->(form) { form.errors.any? }
        }
```

Comment (autotelik) [inline app/forms/flood_risk_engine/steps/correspondence_contact_email_form.rb]: good to know how to reach the form. I don't think its as readable though.

helper is nice idea will add

Comment (woodpigeon) [inline app/forms/flood_risk_engine/steps/correspondence_contact_email_form.rb]: OK - though for other steps I'd be happy if we didn't lean on reform for validation extras - might  cause pain down the line eg if reform API changes and breaks our validations, or we decide switch out reform (its not adding much as it is).

**Comment by woodpigeon** [comment]
:ship: it

**Pull request merged**